plugins/versionpress/bootstrap.php

```diff
@@ -38,6 +38,20 @@
     define('VP_GIT_BINARY', 'git');
 }
 
+if (!defined('VP_GIT_BINARY')) {
```

Comment:
@borekb you are right. It was an oversight. Fixed now c9c777942a9e88efbc58ad60d8b4e717d329fdf1 . Thanks.